cards/zgoo.json

```diff
@@ -0,0 +1,680 @@
+[
+  {
+    "back_flavor": "Bred from birth to take on the mantle of shrine maiden of her village, Yuki endured all manner of grueling 'training' as a young girl in pursuit of her ultimate vocation. Unfortunately, the final rite to 'purify' her soul involved an arcane ritual which went haywire and left Yuki stricken by a sort of trauma-induced amnesia. That fateful night, Yuki fled the village in a panic and managed to steal away to Arkham, where she eventually cultivated a life of service tending to the city's poverty-stricken and dispossessed masses. Her kind and patient demeanor draws the tormented to her like a lodestone, and as her good works multiply so too have the memories of what she left behind started to resurface...",
+    "back_text": "<b>Deck Size</b>: 30.\n<b>Deckbuilding Options</b>: Guardian cards ([guardian]) level 0-5, Mystic cards ([mystic]) level 0-1, <b>Blessed</b> cards level 0-5, Neutral cards level 0-5.\n<b>Deckbuilding Requirements</b> (do not count toward deck size): Apotheosis, The Succession, 1 random basic weakness.\n<b>Deckbuilding restrictions</b>: No [[Weapon]] assets.",
+    "code": "zgoo_00001",
+    "deck_limit": 1,
+    "deck_options": [
+      {
+        "faction": [
+          "guardian"
+        ],
+        "level": {
+          "min": 0,
+          "max": 5
+        }
+      },
+      {
+        "faction": [
+          "mystic"
+        ],
+        "level": {
+          "min": 0,
+          "max": 1
+        }
+      },
+      {
+        "faction": [
+          "neutral"
+        ],
+        "level": {
+          "min": 0,
+          "max": 5
+        }
+      },
+      {
+        "trait": [
+          "blessed"
+        ],
+        "level": {
+          "min": 0,
+          "max": 5
+        }
+      },
+      {
```

Comment:
Not rules need to go first in the list, since if something above it matches it will count (instead of reject)